examples/tutorial/lessons/lesson-01-plumbing.adoc

```diff
@@ -0,0 +1,36 @@
+
+== Lesson 01 - Plumbing
+
+Hawkular Alerting (hAlerting) is a standalone alerting system, written in Java, designed to be embedded into other software products to give them alerting abilities. It is delivered as a WAR file and deployed on a JBoss WildFly application server.  It uses a Cassandra database for persistence. It is already embedded into the Hawkular Metrics project, which is a high volume Time Series Database, built on Cassandra, and used for system monitoring.  You don't need to know much about Cassandra for this tutorial, only that it needs to be running to support hAlerting (and hMetrics for that matter).
+
+If you are comfortable with Maven, Git, WildFly, Cassandra and want to build and run hAlerting, you can certainly do so by following guidelines at the https://github.com/hawkular/hawkular-alerts[hAlerting github site].  But hopefully faster will be to use the docker containers available for running hMetrics and Cassandra.  Remember that hMetrics embeds hAlerting, so it will give us the full alerting system as well as a metrics tool we can use to help demonstrate features.
+
+If you already have Docker running, great,  If not you'll want to https://docs.docker.com/engine/installation/[install Docker now].  Additionally, we use Docker-Compose to combine the hMetrics and Cassandra containers, so you'll want to also https://docs.docker.com/compose/install/[install Docker-Compose now].
+
+If somehow you haven't already, you'll also want to unzip [LINK this tutorial].
+
+When you have docker running you can start the tutorial containers.  Note that the the tutorial will want to use port 8080, so make sure it is free.  Also, if you haven't seen already, running docker will likely need "root" user permission.
+
+`> cd TUTORIAL_HOME/lesson-01`
+`> docker-compose up -d`
+
+This should start two containers, one for the tutorial (hMetrics) and one for Cassandra.  You should see these running with:
+
+`> docker ps`
+
+Let's check to see if things are running via a simple REST request.  In a browser try url http://localhost:8080/hawkular/alerts/status. It should return some JSON, if you see: "status":"STARTED" in the response then we should be good to go.
+
+Because hAlerting is REST based we'll be submitting a lot of REST HTTP requests as part of the tutorial.  You can do this in various ways, such as curl, but a nice tool is the Chrome Advanced REST Client (ARC).  For Chrome users you may want to install this app now via chrome://apps.
+
+The last bit of plumbing is data.  We'll need some incoming data on which we can generate alerts.  To start pumping in some data, run the provided script:
+
+`> ./tutorial-data.sh`
+
+For now, just make sure the script is working before we move on to Lesson 02.  Now that the plumbing is in place we can start building some triggers...
+
+
+Next: Lesson 02 - First Alert
```

Comment:
Cheap one: a relative link to lesson 02 file is a nice to have, so, if I am on github or a local asciidoc viewer I can jump easily.